### PR TITLE
Migrate Planning guide upstream

### PR DESCRIPTION
**Pull request by tahliar**

### CODE REVIEW

File: guides/doc-Planning_Guide/topics/Introduction.adoc [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+[[Supported_Operating_Systems_and_Architectures]]
+=== Supported Operating Systems and Architectures
+
+[[Foreman_Server_Operating_System]]
+==== {ProjectServer} Operating System
+
+{Project} has packages for CentOS 7 and clones, and Debian and clones. CentOS is the recommended platform for the deployment.
```
Comment (lzap) [inline guides/doc-Planning_Guide/topics/Introduction.adoc:166]: > CentOS is the recommended platform for the deployment.

Maybe "Katello plugin packages, which provides content management capabilities, are only available for CentOS.

File: guides/doc-Planning_Guide/topics/Capsule_Server_Overview.adoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+{SmartProxyServer}s provide *content federation* and run *localized services* to discover, provision, control, and configure hosts. You can use {SmartProxies} to extend the {Project} deployment to various geographical locations. This section contains an overview of features that can be enabled on {SmartProxies} as well as their simple classification.
+
+For more information about {SmartProxy} requirements, installation process, and scalability considerations, see link:{BaseURL}installing_capsule_server/[Installing {SmartProxyServer}].
+
+[[sect-Documentation-Architecture_Guide-Capsule_Features]]
+=== {SmartProxy} Features
+
+There are two sets of features provided by {SmartProxyServer}s. You can configure the {SmartProxy} to mirror content from the {ProjectServer}. You can also use the {SmartProxy} to run services required for host management.
+
+Content related features are:
```
Comment (lzap) [inline guides/doc-Planning_Guide/topics/Capsule_Server_Overview.adoc:13]: Content related features, provided by Katello plugin, are: (only upstream, I think it is fair to point this out once again).

File: guides/doc-Planning_Guide/topics/Capsule_Server_Overview.adoc [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+* *Repository synchronization* – the content from the {ProjectServer} (more precisely from selected life cycle environments) is pulled to the {SmartProxyServer} for content delivery (enabled by Pulp).
+
+* *Content delivery* – hosts configured to use the {SmartProxyServer} download content from that {SmartProxy} rather than from the central {ProjectServer} (enabled by Pulp).
+
+* *Host action delivery* – {SmartProxyServer} executes scheduled actions on hosts.
+
+* *Red Hat Subscription Management (RHSM) proxy* – hosts are registered to their associated {SmartProxyServer}s rather than to the central {ProjectServer} or the Red{nbsp}Hat Customer Portal (provided by Candlepin).
+
+Infrastructure and host management services are:
```
Comment (lzap) [inline guides/doc-Planning_Guide/topics/Capsule_Server_Overview.adoc:24]: This should probably come first, then content related features.

**Comment by tahliar** [comment]
Thanks @lzap, I've pushed those changes.

**Comment by tahliar** [comment]
Travis is failing on https://yum.theforeman.org/client/2.0/ but the link works for me.

**Comment by lzap** [comment]
@tahliar @spetrosi can you guys do me a small favor and exclude my name from commit messages? `Add @lzap's feedback` actually causes up to four notifications I need to clear in my GH inbox - when it's pushed, when its merged or backported :-)

Just don't use `@` if you need my name on it... It will be eventually squashed anyway so commit message does not matter much. Cheers!

**Pull request merged**